spec/lib/efile/nc/d400_calculator_spec.rb

```diff
@@ -9,6 +9,55 @@
     )
   end
 
+  describe "Line 10b: Child Deduction" do
+    [
+      [["single", "married_filing_separately"], [
+        [20_000, 6000],
+        [30_000, 5000],
+        [40_000, 4000],
+        [50_000, 3000],
+        [60_000, 2000],
+        [70_000, 1000],
+        [70_001, 0],
+      ]],
+      [["head_of_household"], [
+        [30_000, 6000],
+        [45_000, 5000],
+        [60_000, 4000],
+        [75_000, 3000],
+        [90_000, 2000],
+        [105_000, 1000],
+        [105_001, 0]
+      ]],
+      [["married_filing_jointly", "qualifying_widow"], [
+        [40_000, 6000],
+        [60_000, 5000],
+        [80_000, 4000],
+        [100_000, 3000],
+        [120_000, 2000],
+        [140_000, 1000],
+        [140_001, 0]
+      ]]
+    ].each do |filing_statuses, agis_to_deductions|
+      filing_statuses.each do |filing_status|
+        context "#{filing_status}" do
+          let(:intake) { create(:state_file_nc_intake, filing_status: filing_status, raw_direct_file_data: StateFile::XmlReturnSampleService.new.read("nc_shiloh_hoh")) }
+          let(:calculator_instance) { described_class.new(year: MultiTenantService.statefile.current_tax_year, intake: intake) }
+
+          agis_to_deductions.each do |fagi, deduction_amount|
+            it "returns the value corresponding to #{fagi} FAGI multiplied by number of QCs" do
+              intake.direct_file_data.fed_agi = fagi
+              intake.direct_file_data.qualifying_children_under_age_ssn_count = 2
+
+              calculator_instance.calculate
+              expect(calculator_instance.lines[:NCD400_LINE_10B].value).to eq(deduction_amount)
```

Comment:
[Dust] can we rename the `deduction_amount` to `deduction_amt_for_two_children` to be more explicit about the amount -- I was initially confused that there wasn't `x 2` math going on but realized you had doubled the deduction amounts.